shared/js/project.js

```diff
@@ -127,7 +111,7 @@ module.exports = {
       $('.project_updated_at').text("Not saved"); // TODO i18n
     }
   },
-  appToProjectUrl: function () {
+  appToProjectUrl_: function () {
```

Comment:
Good catch. I did grep for these, but apparently didn't do a very good job of it.